boards/Kconfig

```diff
@@ -1747,6 +1747,7 @@ config ARCH_BOARD_NUCLEO_H745ZI
 	depends on ARCH_CHIP_STM32H745ZI
 	select ARCH_HAVE_LEDS
 	select ARCH_HAVE_BUTTONS
+	select HAS_PWR_DIRECT_SMPS_SUPPLY
```

Comment:
where you define HAS_PWR_DIRECT_SMPS_SUPPLY

Reply:
@xiaoxiang781216 

I added HAS_PWR_DIRECT_SMPS_SUPPLY to stm32h7/Kconfig.
Thank you for your advice.